examples/java/spring-boot/src/main/java/uk/gov/api/springboot/config/MdcFacade.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.api.springboot.config;
+
+import org.slf4j.MDC;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MdcFacade {
```

Comment:
Did we want to `implement MDCAdapter`?

Reply:
I've left this for now as something we can possibly do later in a separate PR